### ISSUE TITLE
How do we record and store the information we collect from services

### ISSUE THREAD

**Issue by kellylee-gds** [alphagov/govuk-design-system#3595]
## What
Agree how to record and store the information we collect from services.

## Why
To ensure it is:
- findable
- navigable
- secure
- ethically managed
- can be expanded

## Who needs to work on this
TBC

## Done when
- [x] explored various options of how we can store information

**Comment by Ciandelle** [comment]
Link to [trello](https://trello.com/b/irGJoNFb/how-do-we-record-and-store-the-information-we-collect-from-services) board where discussion can be held.

**Comment by kellylee-gds** [comment]
Notes from discussion on 04/03/23:

Discussed options — ruled out Google Doc or spreadsheets as advanced macros would be needed to build something to meet our needs. We discussed the option of adapting the Navigation squad's tool which pulls data from a Google doc into a website. However we were concerned about the maintenance overhead of building on this tool and whether we will have the skills in the team to keep it up to date.

We agreed our best option was a paid tool. We discussed both Airtable and Notion. This created two new tasks for this brief:

- add
- add

Kelly & Ollie will create prototypes by the end of the week and when Ciandelle is back on Monday we can make a decision on which to proceed with.
